platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -84,7 +81,7 @@
  * SwirldState}. The Hedera application, after startup, only needs the ability to get {@link
```

Comment:
seems this javadoc needs an update as well

Reply:
Fixed